samples/bluetooth/iso_broadcast_benchmark/src/receiver.c

```diff
@@ -0,0 +1,510 @@
+
+	switch (data->type) {
+	case BT_DATA_NAME_SHORTENED:
+	case BT_DATA_NAME_COMPLETE:
+		len = MIN(data->data_len, DEVICE_NAME_LEN - 1);
+		memcpy(name, data->data, len);
+		name[len] = '\0';
+		return false;
+	default:
+		return true;
+	}
+}
+
+static void scan_recv(const struct bt_le_scan_recv_info *info,
+		      struct net_buf_simple *buf)
+{
+	char le_addr[BT_ADDR_LE_STR_LEN];
+	char name[DEVICE_NAME_LEN];
+
+	if (broadcaster_found) {
+		return;
+	}
+
+	(void)memset(name, 0, sizeof(name));
+
+	bt_data_parse(buf, data_cb, name);
+
+	if (strncmp(DEVICE_NAME, name, strlen(DEVICE_NAME))) {
+		return;
+	}
+
+	bt_addr_le_to_str(info->addr, le_addr, sizeof(le_addr));
+
+	LOG_INF("Found broadcaster with address %s (RSSI %i)",
+		log_strdup(le_addr), info->rssi);
+
+	broadcaster_found = true;
+
+	per_sid = info->sid;
+	per_interval_ms = BT_INTERVAL_TO_MS(info->interval);
+	bt_addr_le_copy(&per_addr, info->addr);
+
+	k_sem_give(&sem_per_adv);
+}
+
+static struct bt_le_scan_cb scan_callbacks = {
+	.recv = scan_recv,
+};
+
+static void sync_cb(struct bt_le_per_adv_sync *sync,
+		    struct bt_le_per_adv_sync_synced_info *info)
+{
+	LOG_INF("Periodic advertisement synced");
+
+	k_sem_give(&sem_per_sync);
+}
+
+static void term_cb(struct bt_le_per_adv_sync *sync,
+		    const struct bt_le_per_adv_sync_term_info *info)
+{
+	LOG_INF("Periodic advertisement sync terminated");
+
+	per_adv_lost = true;
+	k_sem_give(&sem_per_sync_lost);
+}
+
+static void biginfo_cb(struct bt_le_per_adv_sync *sync,
+		       const struct bt_iso_biginfo *biginfo)
+{
+	if (biginfo_received) {
+		return;
+	}
+
+	LOG_INF("BIGinfo received: num_bis %u, nse %u, interval %u ms, "
+		"bn %u, pto %u, irc %u, max_pdu %u, sdu_interval %u us, "
+		"max_sdu %u, phy %s, %s framing, %sencrypted",
+		biginfo->num_bis, biginfo->sub_evt_count,
+		BT_INTERVAL_TO_MS(biginfo->iso_interval),
+		biginfo->burst_number, biginfo->offset, biginfo->rep_count,
+		biginfo->max_pdu, biginfo->sdu_interval, biginfo->max_sdu,
+		phy2str(biginfo->phy), biginfo->framing ? "with" : "without",
+		biginfo->encryption ? "" : "not ");
+
+	iso_interval_ms = BT_INTERVAL_TO_MS(biginfo->iso_interval);
+	bis_count = MIN(biginfo->num_bis, CONFIG_BT_ISO_MAX_CHAN);
+	biginfo_received = true;
+	k_sem_give(&sem_per_big_info);
+}
+
+static struct bt_le_per_adv_sync_cb sync_callbacks = {
+	.synced = sync_cb,
+	.term = term_cb,
+	.biginfo = biginfo_cb,
+};
+
+static void print_stats(char *name, struct iso_recv_stats *stats)
+{
+	/* Convert to a two decimal value without using floats, by
+	 * multiplying with 10000 before division, and then seperating
+	 * the lowest 2 digits
+	 */
+	uint8_t integer;
+	uint8_t fraction;
+	uint32_t total_packets;
+	uint64_t tmp;
+
+	total_packets = stats->iso_recv_count + stats->iso_lost_count;
+	tmp = ((uint64_t)stats->iso_recv_count * 10000) / total_packets;
+
+	integer = tmp / 100;
+	fraction = tmp - integer * 100;
+
+	LOG_INF("%s: Received %u/%u (%u.%02u%%) - Total packets lost %u",
```

Comment:
I considered this as well - It would work well for the first 100k packets, but above that we would run into alignment issues again (and again at 1 million). 

If we want alignment, we should probably do it dynamically and add spaces in a loop. 

Do you want me to add the suggested alignment, leave it be or fully fix the issue?